bootstrap/build.gradle

```
@@ -18,8 +18,13 @@ dependencies {
 
     implementation 'org.springframework.boot:spring-boot-starter-web'
     implementation 'com.github.ulisesbocchio:jasypt-spring-boot-starter:3.0.3'
+    implementation 'org.springframework.boot:spring-boot-starter-data-jpa'
 
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
+
+    // slf4j & logback
+    implementation('org.slf4j:jcl-over-slf4j')
+    implementation('ch.qos.logback:logback-classic')
```

implementation방식을 위와 통일하는 것은 어떨까요?
```suggestion
    // slf4j & logback
    implementation 'org.slf4j:jcl-over-slf4j'
    implementation 'ch.qos.logback:logback-classic'
```